### ISSUE TITLE
Uncaught TypeError: Cannot read property 'calculateCurrentActions' of undefined

### ISSUE THREAD

**Issue by bradbeck** [mulesoft/api-workbench#272]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.5
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package, v0.8.34

### Stack Trace

Uncaught TypeError: Cannot read property 'calculateCurrentActions' of undefined

```
At /Users/bradbeck/.atom/packages/api-workbench/main.js:2

TypeError: Cannot read property 'calculateCurrentActions' of undefined
    at Object.t.calculateItems (/Users/bradbeck/.atom/packages/api-workbench/main.js:2:10077)
    at u (/Users/bradbeck/.atom/packages/api-workbench/main.js:2:8279)
    at atom-workspace.t (/Users/bradbeck/.atom/packages/api-workbench/main.js:2:7833)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
  2x -0:06.4.0 command-palette:toggle (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "excludeVcsIgnoredPaths": false,
    "ignoredNames": [
      ".hg",
      ".svn",
      ".DS_Store"
    ]
  }
}
```

### Installed Packages

```coffee
# User
api-workbench, v0.8.34
asciidoc-preview, v2.5.2
language-asciidoc, v1.5.1
language-groovy, v0.6.0
language-puppet, v0.19.1
language-scala, v1.1.6
linter, v1.11.14
minimap, v4.24.7
pretty-json, v1.4.4
tab-control, v0.6.9
tabs-to-spaces, v1.0.2

# Dev
No dev packages
```

**Comment by ddenisenko** [comment]
@bradbeck  could you please provide reproduction steps for this issue to reproduce?
What were you doing when you got the error?

**Comment by ddenisenko** [comment]
Also, how did you install your copy of API Workbench? Did you install it from Atom->Preferences->Packages or manually?

**Comment by dsteffann** [comment]
I had the same error. I also have Atom 1.8.0 and api-workbench 0.8.34. I got the error while focused in a raml file and trying to open the command palette with ctrl-shift-p. 

I updated my api-workbench through Atom Settings->Updates

**Comment by ddenisenko** [comment]
@dsteffann Could you please check if you have `raml-actions` folder in `/Users/<user_name>/.atom/packages/api-workbench/node_modules` folder?

**Comment by hulkish** [comment]
I'm having same issue as reported, and I do have the `raml-actions` folder as asked:

```bash
#: ls ~/.atom/packages/api-workbench/node_modules -la
- ./
- ../
- dist/
- src/
- typings/
- .npmignore
- .travis.yml
- README.md
- license.txt
- package.json
- tsconfig.json
- typings.json
- workspace.json
```

**Comment by ddenisenko** [comment]
Thanks a lot for reporting the issue. We can reproduced it in case of invoking "Toggle Command Palette". This will be fixed shortly and released today.

Please let me know if you can reproduce this issue in any other use case.

**Comment by ddenisenko** [comment]
Fixed. Fix is published in 0.8.35.

**Comment by dsteffann** [comment]
Confirmed, thank you!!!
